### ISSUE TITLE
Photoprism missing randomly latest photos after update

### ISSUE THREAD

**Issue by Linkandzelda** [photoprism/photoprism#1372]
13 days ago I updated with docker-compose pull to fix a problem where videos were not being downloaded via nextcloud webdav. That issue went away, but ever since I'm missing about half my new pictures. It seems to happen with pictures taken around the same time. Other times just single pictures are completely missing.

I opted to do a reindex of the Originals, and this picked up the missing pictures. I believe this to be a bug because before upgrading this issue never existed and all new pictures were found, to my knowledge, without any errors.

**Comment by lastzero** [comment]
Maybe they got stacked (grouped with other pictures taken at the same time)?

**Comment by Linkandzelda** [comment]
I don't think so. They didn't show up in the main Album section. I had multiple single photos I took on a certain days which didn't show up at all. After a reindex (not full), all of them appeared. Library count went up by 200. At least that means they do get synced into the Originals folder but for some reason not processed?

**Comment by lastzero** [comment]
Maybe hidden or in review? Did you check the logs?

**Comment by graciousgrey** [comment]
@Linkandzelda Any update on this?

**Comment by Linkandzelda** [comment]
The issue still happens, though I've not updated since I last did it. I just did a reindex and it added another 30 photos or so that were previous missed/unprocessed. There's no errors in the logs

**Comment by graciousgrey** [comment]
There might be very rare timing issues during indexing that prevent single files from being indexed. I will close this for now as those images will be catched during the next index process. Our scheduler allows to set up a cron job for indexing (https://dl.photoprism.org/docker/scheduler/).
